frontend/src/routes/(authenticated)/project/[project_code]/+page.ts

```diff
@@ -98,27 +97,8 @@ export async function load(event: PageLoadEvent) {
 
   event.depends(`project:${projectCode}`);
 
```

Comment:
What do you think about leaving this derived store in the load function and changing it to this:
I think this would preserve SSR.
```suggestion
  const lexEntryCount = derived(nonNullableProject, p => {
    if (p.type === ProjectType.FlEx) {
      if (Number.isInteger(p.flexProjectMetadata?.lexEntryCount)) {
        return p.flexProjectMetadata?.lexEntryCount;
      } else {
        return !browser ? undefined : event.fetch(`/api/project/updateLexEntryCount/${p.code}`, {method: 'POST'})
          .then(x => x.text())
          .then(s => parseInt(s))
          .catch(() => 0);
      }
    } else {
      return 0;
    }
  });
```
Note:
- I switched to `nonNullableProject`
- using `Number.isInteger` is beside the point. I just couldn't resist

Reply:
Done in commit 6172b958, where I implemented your suggestion but also removed the lexEntryCount store from the page.svelte file. (Which is why I couldn't just do "commit suggestion" here, because this is only half the commit needed and I didn't want a partial commit in the PR).

Reply:
Moot now that we've removed the fetch from the project page.